src/text-field-base.tsx

```diff
@@ -235,49 +224,61 @@ export const TextFieldBase = React.forwardRef<any, TextFieldBaseProps>(
                     <HelperText
                         error={error}
                         leftText={helperText}
+                        id={helperTextid}
                         rightText={multiline && maxLength ? `${characterCount}/${maxLength}` : undefined}
                     />
                 }
                 multiline={multiline}
                 fullWidth={fullWidth}
                 fieldRef={fieldRef}
                 readOnly={rest.readOnly}
+                dataAttributes={dataAttributes}
             >
                 {startIcon && <div className={styles.startIcon}>{startIcon}</div>}
 
                 {prefix && (
                     <div
                         className={classNames(
                             styles.prefix,
-                            hasLabel ? styles.prefixWithLabel : styles.prefixWithoutLabel
+                            hasLabel ? styles.inputWithLabel : styles.inputWithoutLabel
                         )}
                         style={{
                             opacity: inputState === 'default' ? 0 : 1,
-                            alignSelf: prefixAlignSelf,
                         }}
                     >
                         <Text3 color={vars.colors.textSecondary} regular wordBreak={false}>
```

Comment:
Prefix color should not change in inverse
![image](https://github.com/Telefonica/mistica-web/assets/6722153/1d6df6d8-378f-4ec1-a6f1-5c10ae1cb9c8)

Reply:
fixed